src/tool_writeout.c

```diff
@@ -105,14 +107,84 @@ static const struct variable replacements[]={
   {"scheme", VAR_SCHEME},
   {"stdout", VAR_STDOUT},
   {"stderr", VAR_STDERR},
+  {"content", VAR_CONTENT},
+  {"error", VAR_ERROR},
   {NULL, VAR_NONE}
 };
 
+void copyandshift(char **replace, char *escape)
+{
+  int offset;
```

Comment:
This is used to hold the *length* of the escape string, not really an offset is it? And since it holds the return from strlen() it ought to be a `size_t` and not an `int`...